integrationTests/chainSimulator/staking/delegation_test.go

```diff
@@ -852,3 +853,262 @@ func generateTransaction(sender []byte, nonce uint64, receiver []byte, value *bi
 		Signature: []byte(mockTxSignature),
 	}
 }
+
+// Test description
+//  Test that merging delegation  with whiteListForMerge and
+//  mergeValidatorToDelegationWithWhitelist contracts still works properly
+
+//  Test that their topups will merge too and will be used by auction list computing.
+//
+func TestChainSimulator_MergeDelegation(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	roundDurationInMillis := uint64(6000)
+	roundsPerEpoch := core.OptionalUint64{
+		HasValue: true,
+		Value:    30,
+	}
+
+	// Test steps:
+	//  1. User A: - stake 1 node to have 100 egld more ( or just pick a genesis validator on internal testnets and top it up with 100 egld)
```

Comment:
Wrong spacing

Reply:
Also wrong comments, there is no internal testnet

Reply:
Also, what 100 egld more than what?

Reply:
udpated comments